### ISSUE TITLE
Vorta does not start automatically in Tuxedo OS

### ISSUE THREAD

**Issue by Dev-Paxton** [borgbase/vorta#1990]
### Description

When I reboot my system, vorta does not start.
Reproduce Issue:

1. Create VM https://www.tuxedocomputers.com/de/TUXEDO-OS-in-VirtualBox-VMware-oder-Gnome-Boxes-testen
2. sudo apt update && sudo apt upgrade -y && sudo apt install vorta
3. Launch vorta and click checkbox in misc
4. Reboot
5. Where vorta?

### Reproduction

- [X] I tried to reproduce the issue.
- [X] I was able to reproduce the issue.

### OS

Tuxedo OS 2, KDE

### Version of Vorta

0.8.3

### What did you install Vorta with?

Distribution package

### Version of Borg

1.2.0

### Logs

```pytb
Couldn't find anything about autostart in the logs
```

**Comment by Dev-Paxton** [comment]
Looks like something is wrong with the .desktop file. It works if I set autostart via system settings.

Vorta .desktop file: vora.desktop
[Desktop Entry]
Name=Vorta
GenericName=Backup Software
Exec=vorta --daemonize
Type=Application
Icon=com.borgbase.Vorta
Categories=Utility;Archiving;Qt;
Keywords=borg;
StartupWMClass=vorta
StartupNotify=false
GNOME-Autostart-enabled=true 
X-GNOME-Autostart-Delay=20

System .desktop file: com.borgbase.Vorta.desktop
[Desktop Entry]
Categories=Utility;Archiving;Qt;
Exec=vorta
GenericName=Backup Software
Icon=com.borgbase.Vorta
Keywords=borg;
Name=Vorta
StartupWMClass=vorta
Type=Application

**Comment by m3nu** [comment]
Autostart is different for each system. If you have a reproducible solution we can add it to the docs though.

**Comment by Dev-Paxton** [comment]
Yes, that would be if you add vorta under `systemsettings -> startup & shutdown -> autostart` (for kde)
